### ISSUE TITLE
Wrong version shown in settings

### ISSUE THREAD

**Issue by overheadhunter** [cryptomator/hub#125]
### Please agree to the following

- [X] I have searched [existing issues](https://github.com/cryptomator/hub/issues?q=) for duplicates
- [X] I agree to follow this project's [Code of Conduct](https://github.com/cryptomator/hub/blob/develop/.github/CODE_OF_CONDUCT.md)

### Summary

Hub 1.0.0-beta2 shows up as 1.0.0

### System Setup

```markdown
- Hub: 1.0.0-beta2
- Keycloak: 19.0.1
```

### Steps to Reproduce

1. Install 1.0.0-beta2
2. Go to Settings

### Expected Behavior

Correct version displayed

### Actual Behavior

<img width="1040" alt="Screenshot 2022-08-17 at 09 34 03" src="https://user-images.githubusercontent.com/1204330/185061343-b09a711a-fe7e-443c-b229-73c306755ec6.png">

### Reproducibility

Always

### Anything else?

Probably the version is taken from here?

https://github.com/cryptomator/hub/blob/e7a903cea7fb4d3cb0f0a4c13813d799cde957dc/backend/pom.xml#L7

In that case, we should make sure to `mvn version:set` during tagged CI builds.

**Comment by infeo** [comment]
> Probably the version is taken from here?
Yes, in the `VersionResource` we read out the quarkus property `quarkus.application.version` and it defaults to the mvn project version:
https://quarkus.io/guides/all-config#quarkus-core_quarkus.application.version

**Comment by infeo** [comment]
@overheadhunter A question to not-tagged/dev builds: Should we set a version there?
Currently, the version on the `release/1.0.0` branch is 1.0.0. Every build (locally and on CI) on this branch has version "1.0.0" set, which is confusing.

My idea ist: Append or replace the version with `SNAPSHOT`

**Comment by overheadhunter** [comment]
On a release branch, it is intended to have the "correct" version for the release already, since we don't want a diff when merging from release to main.

We need a `mvn version:set` during tagged builds, though. Same as we do on other repos.
